pkgs/applications/networking/browsers/qutebrowser/default.nix

```
@@ -55,13 +55,17 @@ in mkDerivationWith python3Packages.buildPythonApplication rec {
     docbook_xml_dtd_45 docbook_xsl libxml2 libxslt
   ];
 
-  propagatedBuildInputs = with python3Packages; [
+  propagatedBuildInputs = with python3Packages; ([
     pyyaml backendPackage jinja2 pygments
     pypeg2 cssutils pyopengl attrs setuptools
```

```suggestion
    pyopengl
```
https://github.com/qutebrowser/qutebrowser/blob/master/doc/changelog.asciidoc#v200-2021-01-28

---

PyOpenGL isn't needed either - that dependency got [removed in v1.0.0](https://github.com/qutebrowser/qutebrowser/blob/master/doc/changelog.asciidoc#v100-2017-10-12) back in 2017 :wink:

---

Thanks for the hints!